ui/src/shared/components/TableGraph.js

```diff
@@ -139,16 +156,20 @@ class TableGraph extends Component {
   render() {
     const {sortByColumnIndex, hoveredColumnIndex, hoveredRowIndex} = this.state
     const {hoverTime, tableOptions, colors} = this.props
-    const {data} = this.state
+
+    const verticalTimeAxis = _.get(tableOptions, 'verticalTimeAxis', true)
+
+    const data = verticalTimeAxis ? this.state.data : this.state.unzippedData
```

Comment:
can peal data and unzipped data off of state

Reply:
i didn't do this. :/